mappings/net/minecraft/fluid/FlowableFluid.mapping

```diff
@@ -98,6 +98,16 @@ CLASS net/minecraft/class_3609 net/minecraft/fluid/FlowableFluid
 		ARG 2 pos
 		ARG 3 state
 		ARG 4 fluid
+	METHOD method_15782 (Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;Lnet/minecraft/class_3610;)Lnet/minecraft/class_243;
```

Comment:
Needs a name.

Reply:
I think I fixed it on https://github.com/FabricMC/yarn/pull/3284/commits/5076644a8c3933dec14c80c9f6b0da20b352fffc

I don't see a method with that name on the mappings, but I believe it should be taken care of in that commit

Reply:
```suggestion
```
This line is no longer needed since the docs moved to the parent method.